src/protected/application/lib/modules/Diligence/Controllers/Controller.php

```diff
@@ -243,42 +244,47 @@ public function GET_getAuthorizedProject()
      *
      * @return boolean
      */
-    public function GET_deleteFile() : bool
+    public function GET_deleteFile()
     {
         $app = App::i();
         $conn = $app->em->getConnection();
+
+        $registrationId = $this->data[1];
+        $file = $app->repo('File')->findBy(['id' => $this->data['id']])[0];
+
         //Verificando se existe na rota esse indice
-        if(isset($this->data[1]))
-        {
-            $entity = $app->repo('Registration')->find($this->data[1]);
+        if (isset($registrationId)) {
+            $entity = $app->repo('Registration')->find($registrationId);
             //Verificando se o dono da inscrição é o mesmo usuario logado
-            if($entity->getOwnerUser() == $app->getUser())
-            {
-                $query = $conn->executeQuery('DELETE FROM file WHERE id = '.$this->data['id']);
+            if ($entity->getOwnerUser() == $app->getUser()) {
+                $query = $conn->executeQuery('DELETE FROM file WHERE id = ' . $this->data['id']);
                 $result = $query->execute();
-                if($result)
-                {
+                if ($result) {
```

Comment:
Pra evitar SQL Injection, melhor utilizar bindParam.

```suggestion
                $stmt = $conn->prepare('DELETE FROM file WHERE id = :id');
                $stmt->bindParam('id', $this->data['id']);
                $affectedRows = $stmt->executeStatement();
                
                if ($affectedRows) {
```